keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -310,30 +341,89 @@ required) or even code snippets. If there's any ambiguity about HOW your
 proposal will be implemented, this is the place to discuss them.
 -->
 
-A new optional field `MatchLabelKeys` is introduced to `PodAffinityTerm`.
+A new optional field `MatchLabelSelectors` is introduced to `PodAffinityTerm`.
 
 ```go
+type LabelSelectorOperator string
```

Comment:
Can we define the allowed values? Are you planning only `In` and `NotIn` or sth else too?